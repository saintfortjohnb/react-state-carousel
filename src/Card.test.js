import React from "react";
import renderer from "react-test-renderer";
import { render } from "@testing-library/react";
import Card from "./Card";

it("renders without crashing", function () {
  render(
    <Card
      caption="Test Caption"
      src="test.com/image.jpg"
      currNum={1}
      totalNum={3}
    />
  );
});

it("matches snapshot", function () {
  const tree = renderer
    .create(
      <Card
        caption="Test Caption"
        src="test.com/image.jpg"
        currNum={1}
        totalNum={3}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
