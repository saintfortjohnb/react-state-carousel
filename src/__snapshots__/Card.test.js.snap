// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="Card"
>
  <h4
    className="Card-title"
  >
    Test Caption
  </h4>
  <img
    alt="Test Caption"
    className="Card-image"
    src="test.com/image.jpg"
  />
  <small
    className="Card-small"
  >
    Image 
    1
     of 
    3
    .
  </small>
</div>
`;
